M2Mqtt/Net/MqttNetworkChannel.cs

```diff
@@ -193,7 +197,11 @@ public MqttNetworkChannel(string remoteHostName, int remotePort, bool secure, X5
             // in this case the parameter remoteHostName isn't a valid IP address
             if (remoteIpAddress == null)
             {
+#if NETSTANDARD1_6
+                IPHostEntry hostEntry = Dns.GetHostEntryAsync(remoteHostName).Result;
```

Comment:
What is the difference?
I think this is a longest one. Because "Result" blocks the call to get the value of task and "Wait" do the same.